GraphClient.go

```diff
@@ -132,6 +132,11 @@ func (g *GraphClient) refreshToken() error {
 	return err
 }
 
+// returns client Token
+func (g *GraphClient) GetToken() Token {
```

Comment:
What is this `func` for? Why do you need it? In my opinion the token should not be used outside the `GraphClient` anyway. I just want to understand what you are trying to achieve with this 😃

Reply:
In my app I'm using the token to forward it to another service for authorization. If you want to keep it private that is not a problem but all the necessary element to get the token are there (TennantID, AppID and ClientCredentials) to make an API call and get the token anyway so I think that the token should be accessible to spare unnecessary API calls.